### PR TITLE
Whitepaper buttons

### PR DESCRIPTION
**Pull request by bkase**

### CODE REVIEW

**Comment by bkase** [comment]
Screen Shot 2020-01-15 at 10 50 41 PM

This took longer than I expected, haha

**Comment by bkase** [comment]
font is too large, but we can fix it in post

File: frontend/website2/src/components/svg/EllipticBackground.re [hunk 1]
```diff
@@ -0,0 +1,182 @@
+let svg =
```
Comment (Schmavery) [inline frontend/website2/src/components/svg/EllipticBackground.re]: why not just an svg file?

File: frontend/website2/src/components/WhitepaperButton.re [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  let make = (~sigil) => {
+    <div className=Style.container>
+      <span className=Style.page> Icons.page </span>
+      <span className=Style.sigil> sigil </span>
+    </div>;
+  };
+};
+
+[@react.component]
+let make = (~label, ~sigil) => {
```
Comment (Schmavery) [inline frontend/website2/src/components/WhitepaperButton.re]: ~icon?

**Pull request merged**